gcloud-java-compute/src/test/java/com/google/gcloud/compute/RawImageConfigurationTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.compute;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+
+import com.google.gcloud.compute.ImageConfiguration.SourceType;
+
+import org.junit.Test;
+
+public class RawImageConfigurationTest {
+
+  private static final String SOURCE = "source";
+  private static final SourceType SOURCE_TYPE = SourceType.RAW;
+  private static RawImageConfiguration CONFIGURATION = RawImageConfiguration.builder(SOURCE)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/mziccard/gcloud-java/file/2314353647/issues/source?bid=3140968&fileBranchId=3200309#l32)